internal/commands/upload_release_test.go

```diff
@@ -177,7 +177,9 @@ compiled_packages:
 			BeforeEach(func() {
 				f, err := fs.Create("invalid-release.tgz")
 				_, _ = f.Write([]byte("invalid"))
-				defer func() { f.Close() }()
+				defer func() {
+					_ = f.Close()
```

Comment:
This seems like the appropriate cleanup is `defer f.Close()`.

Reply:
This syntax makes it explicit that the close error is being ignored and makes the code pass the Goland linter. `defer f.Close()` causes an "ignored error" warning.